class_person.rb

```diff
@@ -1,14 +1,16 @@
 require_relative 'nameable'
+require_relative 'rental'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rental
   attr_accessor :name, :age
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rental = []
```

Comment:
- [ ] Great job for adding the rental array here but I think A method for adding rentals should be present so please add it you can check this [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for reference 🎉🔥💯🙏🏻

While creating the Rental 💯 The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association

```
    Rental.new(date, book, self)
```